controllers/requestsController.js

```diff
@@ -10,6 +13,17 @@ class RequestsController {
   //Need to update this function later
   async acceptRequest(req, res) {
     const { beneId, bookId } = req.body;
+    const smsConsent = await this.userModel.findOne({
+      where: { id: beneId, smsConsent: true },
+    });
+
+    const recipientNumber = await this.userModel.findOne({
+      where: { id: beneId },
+      attributes: ["phone"],
+    });
+
+    console.log(req.body);
+    console.log(recipientNumber.phone);
```

Comment:
Please wrap these function inside try block in case there is a error with the connection with DB.